api/backup.go

```diff
@@ -54,6 +109,60 @@ func (s *Server) BackupVolumeGet(w http.ResponseWriter, req *http.Request) error
 	return nil
 }
 
+func (s *Server) BackupVolumeUpdateAll(w http.ResponseWriter, req *http.Request) error {
+	var input BackupVolume
+
+	apiContext := api.GetApiContext(req)
+	if err := apiContext.Read(&input); err != nil {
+		return err
+	}
+
+	bvs, err := s.m.ListBackupVolumesSorted()
+	if err != nil {
+		return errors.Wrap(err, "failed to list backup volumes")
+	}
+
+	if input.SyncBackupVolume {
+		for _, bv := range bvs {
+			if _, err := s.m.SyncBackupVolume(bv); err != nil {
+				logrus.WithError(err).Warnf("Failed to synchronize backup volume %v", bv.Name)
+			}
+		}
+	}
+
+	apiContext.Write(toBackupVolumeCollection(bvs, apiContext))
+	return nil
+}
+
+func (s *Server) SyncBackupVolume(w http.ResponseWriter, req *http.Request) error {
+	var input SyncBackupVolume
```

Comment:
Why not just use `BackupVolume` model to get the volume name instead of creating a new model. The functions related to backup volume should be able to reuse the same request model?

Reply:
Moved to struct SyncBackupResource.
Thanks.